projects/plugins/jetpack/modules/custom-css/csstidy/class.csstidy.php

```diff
@@ -943,7 +943,11 @@ public function parse( $string ) {
 						}
 
 						if ( 'iv' === $this->status ) {
-							if ( ! $_quoted_string ) {
+							// phpcs:disable
```

Comment:
If you specify the specific rules here and on line 960, then line 960 won't override line 32 and you won't have to add lines 990 and 1039.

Specifying the specific rules is also generally better practice IMO.

```php
// phpcs:disable Squiz.PHP.CommentedOutCode.Found, Squiz.Commenting.BlockComment.NoNewLine
...
// phpcs:enable Squiz.PHP.CommentedOutCode.Found, Squiz.Commenting.BlockComment.NoNewLine
```